docs/openapi/howto/archboard_feedback.md

```diff
@@ -0,0 +1,152 @@
+# Archboard DPG feedback loop
+
```

Comment:
As the tile is for DPG, does it also cover the guide for JS RLC?

Reply:
Yes, I can clarify that